### PR TITLE
Create General Filters component

### PR DESCRIPTION
**Pull request by bazelajoanna**
Create SessionFilters component with toggle button to switch between Fixed and Mobile sessions.

image

### CODE REVIEW

File: app/javascript/react/components/Map/Map.tsx [hunk 1]
```diff
@@ -93,15 +91,14 @@ const Map = () => {
   const [selectedSessionId, setSelectedSessionId] = useState<number | null>(
     null
   );
-  const [selectedSessionType, setSelectedSessionType] = useState<SessionType>(
-    SessionTypes.FIXED
-  );
   const [selectedStreamId, setSelectedStreamId] = useState<number | null>(null);
 
+  // Selectors
+  const selectedSessionType = useAppSelector(selectSelectedSessionType);
```
Comment (JoannaThelema) [inline app/javascript/react/components/Map/Map.tsx:97]: I didn't know we have `useAppSelector` 😅 Makes sense to use it everywhere for consistency but this is a note for me 🙈

File: app/javascript/react/components/SessionFilters/SessionFilters.style.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  color: ${(props) => (props.$isActive ? white : gray300)};
+  cursor: pointer;
+
+  &:first-child {
+    border-radius: 10px 0 0 10px;
+  }
+
+  &:nth-child(2) {
+    border-radius: 0 10px 10px 0;
+  }
```
Comment (JoannaThelema) [inline app/javascript/react/components/SessionFilters/SessionFilters.style.tsx:38]: Nice! I didn't know this trick 👍

**Review by JoannaThelema — APPROVED** [review]
To be consistent I would name all icons in the same pattern eg: `pinIcon` WDYT?

Overall, beautiful buttons 😄

**Review by Sonje — APPROVED** [review]

File: app/javascript/react/components/Map/Map.tsx (hunk 1 above)
Comment (bazelajoanna) [reply app/javascript/react/components/Map/Map.tsx:97]: I didn't know either :D I've just noticed that so I decided to use :)

**Pull request merged**